packit_service/service/api/srpm_builds.py

```diff
@@ -0,0 +1,47 @@
+from http import HTTPStatus
+from logging import getLogger
+
+try:
+    from flask_restx import Namespace, Resource
+except ModuleNotFoundError:
+    from flask_restplus import Namespace, Resource
+
+from packit_service.service.api.utils import response_maker
+from packit_service.service.api.parsers import indices, pagination_arguments
+from packit_service.models import SRPMBuildModel
+
+# from packit_service.service.urls import get_srpm_log_url_from_flask
+from flask import url_for
+
+logger = getLogger("packit_service")
+
+ns = Namespace("srpm-builds", description="SRPM builds")
+
+
+@ns.route("")
```

Comment:
is this really meant to be empty?

Reply:
Yes, because this namespace is at /api/srpm-builds we want it to be available at  / here